src/common/low_precision_transformations/tests/mark_dequantization_subgraph_transformation.cpp

```diff
@@ -868,3 +869,113 @@ TEST_F(TransformationTests, KeepDequantizationPrecisionTransformationFolding) {
     auto result = func_comparator(model_ref, model);
     ASSERT_TRUE(result.valid) << result.message;
 }
+
+inline std::shared_ptr<ov::Model> make_gather_model(
+    ov::element::Type data_type,
+    ov::element::Type indices_type,
+    bool use_data_convert,
+    bool use_indices_convert,
+    bool indices_as_param = false)
+{
+    using ov::op::v0::Parameter;
+    using ov::op::v0::Constant;
+    using ov::op::v0::Convert;
+    using ov::op::v8::Gather;
+
+    ov::ParameterVector params;
+
+    std::shared_ptr<ov::Node> data = std::make_shared<Constant>(data_type, ov::Shape{4}, 2);
+    if (use_data_convert)
+        data = std::make_shared<Convert>(data, ov::element::f32);
+
+    std::shared_ptr<ov::Node> indices;
+    if (indices_as_param) {
+        auto param = std::make_shared<Parameter>(indices_type, ov::Shape{3});
+        params.push_back(param);
+        indices = param;
+    } else {
+        indices = std::make_shared<Constant>(indices_type, ov::Shape{3}, 1);
+    }
+
+    if (use_indices_convert)
+        indices = std::make_shared<Convert>(indices, ov::element::i32);
+
+    auto axis = std::make_shared<Constant>(ov::element::i64, ov::Shape{1}, 0);
```

Comment:
```suggestion
    auto axis = std::make_shared<Constant>(element::i64, Shape{1}, 0);
```
Detail(optional): As there is global `using namespace ov;` in this file the ov should not be required.
Or remove this using and keep as it is